tools/istio-iptables/pkg/dependencies/implementation.go

```diff
@@ -15,12 +15,37 @@
 package dependencies
 
 import (
+	"bytes"
 	"fmt"
 	"os"
 	"os/exec"
+	"strconv"
 	"strings"
 )
 
+type xtablesExittype int
+
+// Learn from `xtables_exittype` of iptables.
+// `XTF_ONLY_ONCE`, `XTF_NO_INVERT`, `XTF_BAD_VALUE`, `XTF_ONE_ACTION` will eventually turned out to be a
+// parameter problem with explicit error message. Thus, we do not need to support them here.
+const (
+	// OtherProblem indicates a problem of other type in iptables
+	OtherProblem xtablesExittype = iota + 1
```

Comment:
Since those exit codes are specific to xtables, I would include `XTables` in each constant's name.

Reply:
Agree, renamed. Thanks!